cmd/snapd-apparmor/main_test.go

```diff
@@ -64,8 +64,10 @@ func (s *mainSuite) TestIsContainerWithInternalPolicy(c *C) {
 	c.Assert(snapd_apparmor.IsContainerWithInternalPolicy(), Equals, false)
 
 	// simulate being inside WSL
-	testutil.MockCommand(c, "systemd-detect-virt", "echo wsl")
-	c.Assert(snapd_apparmor.IsContainerWithInternalPolicy(), Equals, true)
+	func() {
+		defer snapd_apparmor.MockWSL(true)()
+		c.Assert(snapd_apparmor.IsContainerWithInternalPolicy(), Equals, true)
+	}()
```

Comment:
No need to run this in a function, we can just call restore after the check
```suggestion
	restore := snapd_apparmor.MockWSL(true)
	c.Assert(snapd_apparmor.IsContainerWithInternalPolicy(), Equals, true)
	restore()
```